back/src/database/ReviewSQL.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+const con = require('./DBHandler.js');
+const {info, debug, warning, error}  = require('../winston');
+const { connect } = require('../route/reviewRoute.js');
+const pool = con.pool;
+const fileLabel = "ReviewSQL"
+const Review = require('../objects/review');
+const util = require('util');
+
+module.exports.saveReview = async function(newReview){
+    const checkQuery = {
+        text: 'SELECT spot_id from spots.spots Where spot_id = $1',
+        values: [newReview.spotId]
+    };
+    const query = {
+        text: 'INSERT INTO spots.review(review_id, spot_id, comment, score, user_id) VALUES($1, $2, $3, $4, $5);',
+        values: [newReview.reviewId, newReview.spotId, newReview.comment, newReview.score, newReview.userId]
+    };
+    const client = await pool.connect();
+    return client.query(checkQuery)
+    .then( nullchecker =>{
+        if(nullchecker.rowCount == 0){
+            client.release();
+            error(fileLabel,"There is no spot at: " + newReview.spotId);
+            return {"success":false,"data":newReview.spotId};
+        }
+        return client.query(query)
+        .then(() =>{
+            client.release();
+            debug(fileLabel,"saved review: " + newReview);
```

Comment:
a review is an object, you need to use util.inspect to view it when we when to log in terminal. Otherwise it will only show [Object object]